### DISCUSSION TITLE
The Senate

### DISCUSSION THREAD

**Discussion by michalptacnik** [liberland/constitution#585] [Ideas]
_Quis custodiet ipsos custodes?_ Decimus Iunius Iuvenalis: Satires

This discussion exists to advance the following questions:
* Is the Senate sufficiently powerful, or should it become less or more powerful?
* Should the people have certain powers to censor the Senate or individual Senators where those should become clearly corrupt or acting maliciously?

Please note that Senators are **NOT** immune to prosecution, so we have a way to deal with criminal Senators. They can be slashed off their Pooled LLM, and the Senators may mandate upon themselves to disfellowship any Senator convicted of willful, violent crimes or corruption crimes against Liberland or somesuch. 
Question: Should we mandate them to do so? Or is this hands-off approach where the initiative and self-governance rests with the senator the best way to fulfil Prof. Petrovic's vision for a "Fourth Power of the Government", that of Independent Oversight?

**Comment by JonathanMcCormickJr** [comment]
My main concern with the proposed Senate as I currently understand it is the potential for the Senate to become a tool for the concentration of possible government financial dividends at the expense of citizens in general, resulting in a sort of "socialism-for-the-rich" scenario.

Why am I worried about this? In the Liberland Wiki (https://liberland-1.gitbook.io/wiki/v/public-documents/primers/senate#senator-share) it mentions the following:
"Senators [have] preferential access to potential dividends from the State treasury. In instances where the budget yields a surplus that is deemed excessive for reinvestment in the nation's developmental aspirations, such surplus is earmarked for distribution among the citizenry. However, Senators are accorded a unique privilege in this context, granting them access to a portion of the surplus, proportional to their undistributed share in Liberland Merits (LLM), prior to the allocation to the general populace."

How would this look in practice? Would this simply be a matter of "_The Treasury writes the Senators' checks first, but everyone still gets only what their LLM holdings entitle them to, regardless of whether or not they are a Senator_", or would it be more like, "_The Treasury writes bigger checks to Senators compared to non-Senators of equal LLM holdings_"?

If the latter is the case (where Senators are rewarded more dividends per LLM than regular Citizens), then I think there must be limits in place to prevent abuse in the form of the Senate taking an exorbitant percentage of the total dividend payout.
